bodyfetcher.py

```diff
@@ -208,10 +241,37 @@ def make_api_call_for_site(self, site):
                 owner_rep = 0
             q_id = str(post["question_id"])
 
-            is_spam, reason, why = check_if_spam(title, body, owner_name, owner_link, site, q_id, False, False, owner_rep, post_score)
+            is_spam, reason, why = check_if_spam(
```

Comment:
On the other hand, seeing these on separate lines suggests further improvement (for the future): naming these arguments. It's a long list with some "False, False" in the middle, and the only way to know what these do is to read the code in another module.